source/user-manual/reference/ossec-conf/labels.rst

```diff
@@ -29,21 +29,25 @@ This option specifies the additional information that will appear in alerts. Lab
 
 Attributes:
 
-+--------------------+-------------------------------------------------------------+
-| **key**            | The title that will describe the information of the label.  |
-+                    +---------------------------------------+---------------------+
-|                    | Allowed value                         | Any string          |
-+--------------------+---------------------------------------+---------------------+
-| **hidden**         | For labels that are hidden by default.                      |
-+                    +---------------------------------------+---------------------+
-|                    | Default value                         | no                  |
-+                    +---------------------------------------+---------------------+
-|                    | Allowed value                         | yes,no              |
-+--------------------+---------------------------------------+---------------------+
++--------------------+------------------------------------------------------------------------------------------+
+| **key**            | The title that will describe the information of the label.                               |
++                    +---------------------------------------+--------------------------------------------------+
+|                    | Allowed value                         | Any string that doesn't start with low bar ( _ ) |
++--------------------+---------------------------------------+--------------------------------------------------+
+| **hidden**         | For labels that are hidden by default.                                                   |
++                    +---------------------------------------+--------------------------------------------------+
+|                    | Default value                         | no                                               |
++                    +---------------------------------------+--------------------------------------------------+
+|                    | Allowed value                         | yes,no                                           |
++--------------------+---------------------------------------+--------------------------------------------------+
 
 .. note::
     In ``internal_options.conf``, hidden labels can be set to be displayed in alerts.
 
+.. note::
+    .. versionadded:: 3.9.0
+    The keys started with the character ``_`` are reserved to the system labels. System labels are invisible labels which contain internal information for an agent.
```

Comment:
The keys starting with character ``_`` are reserved to the system labels. This labels are invisible and contain agents internal information.